internal/host/refresh_status_preprocessor_test.go

```diff
@@ -114,7 +114,7 @@ var _ = Describe("Cluster Refresh Status Preprocessor", func() {
 		cluster.Hosts = []*models.Host{&hostToAdd}
 		cluster.LastInstallationPreparation = models.LastInstallationPreparation{
 			Status: models.LastInstallationPreparationStatusFailed,
-			Reason: "",
+			Reason: "There was a failure during preparation",
```

Comment:
why do you add this description? I didn't see that you use it in the PR

Reply:
An empty string never gets stored. If you want the field to change you have to give it a value.
The "Reason" field is a "Free text" field when the status is `LastInstallationPreparationStatusFailed`
This is to allow a detailed error message to be stored. 

In this particular case, it only matters that the field is present, not it's value as we don't use it in any test case apart from one which is covered elsewhere that deals with installation preparation failures.

Reply:
Yes, looking at this specific test, I don't see how we use `LastInstallationPreparation` at all,
I will remove this altogether.